region/maridia/outer.json

```diff
@@ -3876,12 +3876,19 @@
                     "canSuitlessMaridia",
                     "canPreciseWalljump",
                     "canPrepareForNextRoom",
+                    {"or":[
```

Comment:
Sorry, I meant I now think both `canCarefulJump` and `canMidairWiggle` should be required.

Reply:
You were clear, that was my bad

Reply:
Whoops sorry, I guess I'm a little out of it today. Hopefully this has all the necessary information now.